docs/bootstrap.md

```diff
@@ -2,14 +2,14 @@
 title: Bootstrapping
 ---
 
-After creating our resolvers and types classes with other business-related code, we need to make our app running. At first we have to build the schema, then we can expose it by HTTP server, WebSockets or even MQTT.
+After creating our resolvers, types classes, and other business-related code, we need to make our app run. First we have to build the schema, then we can expose it by HTTP server, WebSockets or even MQTT.
 
-## Create executable schema
+## Create Executable Schema
 
-To create executable schema from types' and resolvers' definitions, you need to use `buildSchema` function.
-It takes configuration object as a parameter and returns a promise of `GraphQLSchema` object.
+To create an executable schema from types and resolver definitions, you need to use the `buildSchema` function.
```

Comment:
There are multiple type classes and resolver classes, so I think that it should be "from type and resolver definitions"? 😕